db/migrate/20190102161442_add_taggings_for_all_taggable_tables.rb

```diff
@@ -0,0 +1,56 @@
+class AddTaggingsForAllTaggableTables < ActiveRecord::Migration[5.1]
+  def change
+    create_table "tags", id: :serial, force: :cascade do |t|
+      t.references :tenant, :type => :bigint, :index => false, :null => false, :foreign_key => {:on_delete => :cascade}
+
+      t.string "name", :null => false
+      t.text "description"
+      t.boolean "active", :default => true, :null => false
+
+      # TODO what is human readable name? Will we use it for searching? How will we search with translated field?
+      t.string "display_name"
```

Comment:
Not sure if we need to keep the human readable name in our own datastore?  The IPP states this lives with the tagging taxonomy service.  It would be helpful for our own display needs but if it is owned by that service and is changed we would have to keep them in sync.  Maybe for display we would check with the tagging taxonomy service for a display name based on the tag.

Reply:
@Fryguy @agrare Well it's unclear what the display name should do, if I see display name: ```Deployment Environment```, with name: ```env```. I am gonna search for it using the display name, because that is what I see. But then, I'll see a translation, so I'll search using that translation, right?

And if we'll support starts-with, contains, etc. then we can't do some simple mapping back to name.

So any idea how this should work?

Reply:
I'm not sure if we need to support query by human readable name, the IPP doesn't state but I imagined we would only query based on the tag name

Reply:
Ah, so it's more like what you see as a user. If you see tag ```Deployment Environment``` in french. You'll be filtering it using that. But yeah, it just won't work, so we should ditch the display name and make sure users can see the tag name, so they can filter using that